src/main/ptapi/io/piuio/lxio/lxio.c

```diff
@@ -94,38 +94,6 @@ void convert_output_to_lxio()
 
   /* Menu Buttons */
 
-  // this should be player 2, but its player 1
```

Comment:
That change is actually unrelated to the commit. Can you carve that out and sqash into your lxio commit?